Package.swift

```diff
@@ -19,7 +19,7 @@ import PackageDescription
 
 let package = Package(
   name: "AppCheck",
-  platforms: [.iOS(.v12), .macCatalyst(.v15), .macOS(.v10_15), .tvOS(.v13), .watchOS(.v7)],
+  platforms: [.iOS(.v12), .macCatalyst(.v13), .macOS(.v10_15), .tvOS(.v13), .watchOS(.v7)],
```

Comment:
Based on my understanding, we could reduce this to `macCatalyst(––>.v12<––)`, but I checked the last 10.x release and we had it at `.v13`. I think since we never got support requests for < v13 support, we could keep it as such.

Reply:
I believe Catalyst was introduced at 13